### PR TITLE
Contact me

### PR DESCRIPTION
**Pull request by emarkees**
Adding the Contact me section to the portfolio page, 

I had to create a new section on the html file, added a form.
I gave it a  style with CSS following the Figma template that **[Microverse]**  provided.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -276,5 +276,25 @@ <h3 class="drp_text">Skills</h3>
     </div>
 
   </section>
+  <section class="contactMe">
+      <h1 class="contactTxt">Contact me</h1>
+      <p 
+        class="Contact_body_text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+      </p>
+      <form class="	form" action="https://formspree.io/f/xnqrezql" method="post" >
```
Comment (amejid) [inline index.html:284]: - [ ] Nice job integrating the form with the Formspree service. :+1: Let's work on some minor details to achieve the project requirements. 
- Kindly put placeholder text in your inputs to let visitors know what to put there. ✍️
- Please add some more vertical padding on your inputs to match the design. 🧐️
- Currently, the submit button is not aligned relative to the inputs. Kindly align the button to the left. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- HTML validations working :heavy_check_mark: 
- Formspree service integrated :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -276,5 +276,25 @@ <h3 class="drp_text">Skills</h3>
...
+        <input type="text" id="name" name="user_name" placeholder="FullName"  required maxlength="30"/>
+        <input type="email" id="mail" name="user_email" placeholder="Email Address"  pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-])*$/" required />
+        <div class="conter">
+          <textarea id="textarea" cols="30" rows="10" placeholder="write something here" name="user_message" required maxlength="500"></textarea>
+        </div>
+        <div id="actin">
+          <button type="submit" class="normalButtnAbout" onclick=""> Get in Touch</button>
+        </div>
```
Comment (Ibizugbe) [inline index.html]: Nice work adding `placeholders` to your input fields as requested by the previous reviewer😉 But just as the [previous reviewer suggested](https://github.com/emarkees/Portfolio/pull/3#discussion_r1114606842), let's make our project look more like the design😉 Kindly take note of the i[mages provided by the previous reviewer](https://github.com/emarkees/Portfolio/pull/3#discussion_r1114606842) and make comparison😉 Take note of the `height` and `width` of each input fields and the position of the button😉😉

your form

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @emarkees 👋👋, 

tryagain


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅

- [x] Linters are passing👍
- [x] Github flow has been used👍


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_no optional suggestion_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @emarkees 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**